src/main/java/com/nextstep/lotto/domain/Lottos.java

```diff
@@ -17,25 +17,21 @@ public int count() {
         return lottos.size();
     }
 
-    public LottoStatistics getStatistics(String winningNumberString) {
-        List<Integer> winningNumbers = parseToList(winningNumberString);
-        LottoStatistics lottoStatistics = new LottoStatistics();
+    public LottoStatistics getStatistics(String winningNumberString, int bonusNumber) {
```

Comment:
winningNumberString를 문자열 그대로 받지 말고 InputView에서 파싱해준 리스트로 받는게 좋겠네요
지금은 콘솔 환경이라 문자열로 해도 문제 없지만 웹이나 모바일로 확장했을 때는 입력 방식이 달라질테니까요

```suggestion
    public LottoStatistics getStatistics(List<LottoNumber> winningNumbers, int bonusNumber) {
```

Reply:
다른 객체와 결을 맞춰서 List<Integer>로 받아서 처리해도 괜찮겠네요